src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/HmcHearingsEventTopicListener.java

```diff
@@ -65,23 +66,30 @@ public void onMessage(JmsBytesMessage message) throws JMSException, HmcEventProc
                 Long caseId = hmcMessage.getCaseId();
                 String hearingId = hmcMessage.getHearingId();
 
-                log.info("Attempting to process message from HMC hearings topic for event {}, Case ID {}, and Hearing ID {}.",
-                    hmcMessage.getHearingUpdate().getHmcStatus(), caseId, hearingId);
+                log.info(
+                    "Attempting to process message from HMC hearings topic for event {}, Case ID {}, and Hearing ID {}.",
+                    hmcMessage.getHearingUpdate().getHmcStatus(),
+                    caseId,
+                    hearingId
+                );
 
                 processHmcMessageService.processEventMessage(hmcMessage);
             }
-        }  catch (JsonProcessingException | CaseException | MessageProcessingException ex) {
-            throw new HmcEventProcessingException(String.format("Unable to successfully deliver HMC message: %s",
-                convertedMessage), ex);
+        } catch (JsonProcessingException | CaseException | MessageProcessingException ex) {
+            throw new HmcEventProcessingException(String.format(
+                "Unable to successfully deliver HMC message: %s",
+                convertedMessage
+            ), ex);
         }
+
     }
 
     private boolean isMessageRelevantForService(HmcMessage hmcMessage) {
         return sscsServiceCode.equals(hmcMessage.getHmctsServiceCode());
     }
 
     private boolean isMessageReleventForDeployment(JmsBytesMessage message) throws JMSException {
-        return hmctsDeploymentId == null
+        return hmctsDeploymentId.isEmpty()
```

Comment:
are all the different permutations of this method covered in the test?

Reply:
Have added two more tests to cover the possible scenarios, now covers:
- message and service id match
- message and service id dont match
- empty service id and empty message id 
- empty service id but message id